.gitignore

```diff
@@ -78,4 +78,5 @@ report.asciidoc
 # Bazel
 bazel
 bazel-*
+!/src/dev/bazel
```

Comment:
How about instead put slashes before the two previous lines to make them absolute?

Reply:
👍